02.create-and-explore/03.start-working-in-your-sandbox/02.server-settings-for-your-domain/docs.md

```diff
@@ -34,17 +34,17 @@ This opens the server settings page ([**http://localhost:40100**](http://localho
 
 ![](\server-settings.png)
 
-You can navigate to specific settings using the navbar. By default, only the basic settings can be viewed. To see advanced settings, click **Show Advanced**.
+You can navigate to specific settings Content and Settings tabs. By default, only the basic settings can be viewed. To see advanced settings, click the **Advanced Settings** dropdown beneath each option.
```

Comment:
this first sentence reads strange.   "You can navigate to specific settings Content and Settings Tabs"  What are you trying to say exactly there?